src/TodoList.jsx

```diff
@@ -0,0 +1,28 @@
+
+const todoList = [
+    {
+      id: 1,
+      title: "do coding homework",
+    },
+    {
+      id: 2,
+      title: "Pack for my trip",
+    },
+    {
+    id: 3,
+    title: "watch Udemy videos",
+    }
+  ];
+
+function TodoList() {
+    return (
+        <>
```

Comment:
same comment, no need for the fragment here.

Reply:
ok